### PR TITLE
Fix GA events when using GTM

### PR DESCRIPTION
**Pull request by jonasfj**

### CODE REVIEW

File: static/js/gtm.js [hunk 1]
```diff
@@ -8,38 +8,20 @@ window.dataLayer.push({
...
-    // https://support.google.com/analytics/answer/7478520?hl=en
-    if (elem.hasAttribute('href')) {
-      var updated = false;
-      var callbackFn = function () {
-        if (updated) return;
-        updated = true;
-        document.location = elem.href;
-      };
-      data.event_callback = callbackFn;
-      data.transport_type = 'beacon';
```
Comment (isoos) [inline static/js/gtm.js:35]: Removing this will break the tracking on links, wouldn't it?

Comment (jonasfj) [reply static/js/gtm.js:35]: Yeah, I think it might... but that doesn't work anymore regardless..

Comment (jonasfj) [reply static/js/gtm.js:35]: yes

**Review by isoos — APPROVED** [review]
You'll need to rebase to fix the versions test too.

**Review by isoos — APPROVED** [review]

**Pull request merged**